### PR TITLE
Adding es/2018 01 jo/04

### PR DESCRIPTION
**Pull request by clope031**

### CODE REVIEW

File: src/es/2018-01-jo/04/01.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
-title:  Lección
+title:  La mayordomía... ¿Es más que el dinero?
 date:   20/01/2018
 ---
 
-### <center>Todavía estamos trabajando en esta lección. Por favor, vuelva más tarde.</center>
\ No newline at end of file
+>“En el día de la ira de nada sirve ser rico, pero la justicia libra de la muerte [...] El que confía en sus riquezas se marchita, pero el justo se renueva como el follaje” (Proverbios 11:4, 28).
+
+**Introducción > Salmo 50:10; 119:11**
```
Comment (VitalikL) [inline src/es/2018-01-jo/04/01.md]: @clope031 I would leave Bible verses without making it bold.

File: src/es/2018-01-jo/04/02.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+Piensa en esto por un minuto. ¿Qué sería limpiar una casa de 185 m2en comparación con una de 65 m2? Cuando lo desglosas en fracciones de la vida diaria, estás obteniendo mucho más tiempo para hacer lo que es importante. Esto también significa que el estrés que experimentas al ocuparte de una casa y todo lo que hay en ella -además de estar al tanto de las modas- desaparece casi por completo. Imagina la paz mental y la libertad que obtienes al manejar muchas menos cosas que no son esenciales. 
+
+Como mayordomos del mundo de Dios, debemos seguir este patrón. Al dejar de lado el deseo de ganancias materiales, nos liberamos para pasar más tiempo con él y con nuestro prójimo. Santiago 1:21 dice: “Por esto, despójense de toda inmundicia y de la maldad que tanto abunda, para que puedan recibir con humildad la palabra sembrada en ustedes, la cual tiene poder para salvarles la vida". A veces, al luchar por obtener el éxito material, perdemos de vista el objetivo final. Es mucho más importante conocer a Dios y servirlo que obtener lujos y comodidades mundanales. 
+
+### Para Pensar y Debatir
```
Comment (VitalikL) [inline src/es/2018-01-jo/04/02.md]: I think doing it bold i.e `**` as opposed to `###` would make it better @clope031

File: src/es/2018-01-jo/04/03.md [hunk 3]
```diff
@@ -1,6 +1,46 @@
...
 date:   22/01/2018
 ---
 
-### <center>Todavía estamos trabajando en esta lección. Por favor, vuelva más tarde.</center>
\ No newline at end of file
+**Logos > Proverbios 24:13, 14; Ezequiel 36:26; Mateo 7:7-11; Marcos 11:24; Juan 20:31; 1 Corintios 3:18-20; Gálatas 3:28; Colosenses 3:2; 2 Timoteo 3:16**
+
+Los cristianos debemos ser diferentes de la gente del mundo. Mientras que otros buscan servirse a sí mismos, nosotros deberíamos sacrificarnos. Mientras que el mundo está enfocado en las ganancias materiales, nosotros tendríamos que enfocarnos en las cosas de Dios y en lo que vendrá. Sin embargo, esto va directamente en contra de la condición humana pecaminosa. En su misericordia y gracia, Dios nos ha dado una salida para escapar y dejar atrás las costumbres encarceladoras del mundo. 
+
+### Levanta la vista (Gálatas 3:28; Colosenses 3:2)
```
Comment (VitalikL) [inline src/es/2018-01-jo/04/03.md]: @clope031 And here as well

File: src/es/2018-01-jo/04/04.md [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+
```
Comment (VitalikL) [inline src/es/2018-01-jo/04/04.md]: Trailing `*` above makes all below text italics. @clope031

Comment (clope031) [reply src/es/2018-01-jo/04/04.md]: Changes Made

**Pull request merged**